src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -75,6 +75,17 @@ const FindOnShelfModal: FC<FindOnShelfModalProps> = ({
     );
   });
 
+  // Sorting of the final data below.
```

Comment:
maybe a comment, cause this is not very selfexplanatory. why "00"?

Reply:
Very good. I am being more specific in the comments in the components now :) Thanks!